### PR TITLE
test: setupNetworkWatcher method

### PR DESCRIPTION
**Pull request by suyashpatil78**

### CODE REVIEW

**Comment by suyashpatil78** [comment]
Screenshot 2023-04-06 at 10 24 47 AM

**Review by jayfyle — COMMENTED** [review]
minor comments

**Review by jayfyle — APPROVED** [review]
Looks good to me

File: src/app/fyle/my-profile/my-profile.page.spec.ts [hunk 1]
```diff
@@ -1,26 +1,145 @@
...
+    networkService = TestBed.inject(NetworkService) as jasmine.SpyObj<NetworkService>;
+    orgSettingsService = TestBed.inject(OrgSettingsService) as jasmine.SpyObj<OrgSettingsService>;
+    fixture.detectChanges();
+  }));
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should update isConnected inside setupNetworkWatcher', () => {
```
Comment (Julias0) [inline src/app/fyle/my-profile/my-profile.page.spec.ts:128]: Change in the original variable
`this.networkService.connectivityWatcher(networkWatcherEmitter);`
->
`networkWatcherEmitter = this.networkService.connectivityWatcher(new EventEmitter<boolean>());`

**Pull request closed without merging**